### PR TITLE
New Ancient Med (READY FOR PULL)

### PR DESCRIPTION
**Pull request by JorenC**
I think it should be good. 

@tttppp _when you have time_ can you check and let me know?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,3 +26,4 @@ classical/droidippy/games/*.txt
 test_output_maps
 generator/*.go
 generator/*map.svg
+variants/ancientmediterranean/svg/flags + units.txt
```
Comment (tttppp) [inline .gitignore:29]: If you want to add a new type of file to this then please make it a bit more generic.  e.g. Call the file dev-info.txt or something.

**Comment by JorenC** [comment]
Sorry, I added this as a gitignore thought it wouldn't be added to your
base - it's just a temp file I made to remind me I still need to do this.
I'll remove it!

**Comment by JorenC** [comment]
Or better, please discard that change and leave it. It's an empty file

**Comment by tttppp** [comment]
> Or better, please discard that change and leave it. It's an empty file
I'm not sure I understand what you mean by this. Please can you change the file to however you want it to look, and then if I think it's weird I'll let you know.

**Comment by JorenC** [comment]
Changed it. 
- I meant that I added this file as a personal note.
- I added it to gitignore thinking it would not be added to your end.
- I removed the line, but I figured writing the above comment was probably just as much effort ;)
Now it should be all good.

**Comment by tttppp** [comment]
If you ever want to do this without affecting anyone else then create a file at `.git/info/exclude` with the filename in. It's exactly like the`.gitignore` file but it's not committed (ie it's just for you).

**Comment by JorenC** [comment]
@tttppp this should now be fixed as well.
I made this as a 'generic' old style ship, I might update it later in the future but for now it's 1000x better than what it was.

This one should now also be good to go?

**Comment by tttppp** [comment]
This seems to have changes to the hundred map too. Is that deliberate? Is it ready as well?

**Comment by JorenC** [comment]
There were no real changes, some spaces added to the Hundred.go by my autoformatting, I believe. It's safe to push, but I reverted them anyway.

**Comment by tttppp** [comment]
Thanks for reverting those changes.  The only issue(s) I spotted:
1. Thebes looks like it has two coasts in the new map (it only has one)
2. Related - the Nile is really hard to spot.

**Comment by JorenC** [comment]
@tttppp I've finalized this now,can you pull it?

**Comment by tttppp** [comment]
Can you check the "Files changed" tab as it looks like there are changes to a couple of other variants in this branch. I don't think it's ready for merging anyway.

**Comment by tttppp** [comment]
Something's wrong here - there's a one way arrow.

image

I don't think you intended to change anything about the connections, so maybe this issue already existed?

**Comment by tttppp** [comment]
I've fixed the issue (it was present before your change) and made some minor updates to the unit positions (to avoid overlapping arrows).  I lifted the files without using your commit history, because there were a lot of other changes on your master branch.  If you want the other (non-ancient med) changes then please can you put them on branches of their own?

**Pull request closed without merging**